bash_completion

```diff
@@ -85,7 +85,12 @@ __nvm() {
 # ZSH, load and run bashcompinit before calling the complete function.
 if [[ -n ${ZSH_VERSION-} ]]; then
   autoload -U +X bashcompinit && bashcompinit
-  autoload -U +X compinit && compinit
+  autoload -U +X compinit
+  if [[ $ZSH_DISABLE_COMPFIX = true ]]; then
```

Comment:
this loses the semantic that `compinit` only runs if `autoload` succeeds

Reply:
Committed a change to fix that.